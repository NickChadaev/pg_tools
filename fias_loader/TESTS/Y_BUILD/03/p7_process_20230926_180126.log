2023-09-26 18:01:26 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_03 postgres /home/n.chadaev@abrr.local/Y_BUILD/03/stage_gar_c_03.csv 223-09-25
--------------------------------------------------------
[18:01:26] ... Бурятия Респ, id_region: 3, Execute: /home/n.chadaev@abrr.local/Y_BUILD/03/stage_gar_c_03.csv, process_id:5456
[18:01:26] ... Start process
[18:01:26] ...  -- Version
[18:01:26] ...  -- Очистка от данных
[18:01:26] ...  -- Set UNLOGGED
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_APARTMENT_TYPES_20230925_dec7f8ad-4238-4484-bf99-113a09c762f5.XML
[18:01:26] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDR_OBJ_TYPES_20230925_4f714402-bdf0-4b03-8e2f-cda2704fb403.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ROOM_TYPES_20230925_6fedd268-7191-415b-be0e-8a5a110f7099.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OPERATION_TYPES_20230925_d65578ac-f625-4e50-ac1b-06ee9ac4de7a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_PARAM_TYPES_20230925_72393625-9278-4105-b37d-79dc852810ae.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_HOUSE_TYPES_20230925_1bdc09e7-b8f0-40d5-80f3-cf003785714b.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_ADDHOUSE_TYPES_20230925_cf9caa45-fb3f-40d3-8ddf-770f536de4ed.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_OBJECT_LEVELS_20230925_8ba71762-ad23-4052-b0b8-a659eac50bbb.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_TYPES_20230925_acf1cb3d-a66a-4e4b-b233-cc9c4032d98a.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/AS_NORMATIVE_DOCS_KINDS_20230925_a8475f50-3b47-4725-9f46-b1bd99ea576c.XML
[18:01:27] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_REESTR_OBJECTS_20230925_8e74f1af-2d1e-4ffd-a90e-f87c85f6fd22.XML
[18:10:40] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_ADDR_OBJ_20230925_05d5ef68-06f1-40ea-bd3d-38cc2a3e1c79.XML
[18:11:05] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_HOUSES_20230925_3cac2c24-93ca-4111-bf23-0ac48d1c4ab4.XML
[18:19:54] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_STEADS_20230925_b3827343-3234-4d47-a368-b83fdb533c0a.XML
[18:23:03] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_APARTMENTS_20230925_bb2c387c-2f5d-4840-bd87-088ce5bd380f.XML
[18:25:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_ROOMS_20230925_30b145c5-12d2-49b6-9030-3ef55f7c921b.XML
[18:25:42] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_CARPLACES_20230925_9523ed11-c4a9-421e-9982-4ac978cfadc4.XML
[18:25:43] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_ADM_HIERARCHY_20230925_75317eb9-4709-4584-b719-f2b774ddc8bc.XML
[18:36:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_MUN_HIERARCHY_20230925_65e7914c-f45e-4dc1-a068-bbd5f031c951.XML
[18:46:06] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_CHANGE_HISTORY_20230925_71359a1d-fdb1-4c10-a19f-0d63679b23fb.XML
[19:18:06] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_ADDR_OBJ_DIVISION_20230925_81155461-b7ba-484c-b40b-51afe468e525.XML
[19:18:06] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_ADDR_OBJ_PARAMS_20230925_2fd72bc7-f5fd-4cd9-b6a8-b1c20cf6322a.XML
[19:22:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_STEADS_PARAMS_20230925_bf75d806-c6eb-4fdf-834d-4f9ac9eb4e79.XML
[20:00:50] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_HOUSES_PARAMS_20230925_46f78c9d-4814-47c2-b66d-42210a8f6c50.XML
[20:49:05] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_APARTMENTS_PARAMS_20230925_84128e5b-eafd-4464-819d-fd04513f0f9b.XML
[20:56:48] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_ROOMS_PARAMS_20230925_a2c7e5f8-5fc4-43a8-8109-9f8173e9e017.XML
[20:56:48] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_CARPLACES_PARAMS_20230925_3347d217-31e9-44dd-a22b-2e726bea8bb7.XML
[20:56:49] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_09_21/03/AS_NORMATIVE_DOCS_20230925_77d29d34-3f54-4254-8451-337e1f50307c.XML
[20:59:17] ...  -- Data finish
[20:59:17] ...  -- Set LOGGED
[20:59:50] ... Stop process
--------------------------------------------------------
