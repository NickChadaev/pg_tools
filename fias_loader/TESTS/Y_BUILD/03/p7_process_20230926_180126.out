 save_gar_version 
------------------
               73
(1 row)

CALL
CALL
 set_gar_dt_create 
-------------------
                73
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 744725
gar_fias.as_addr_obj: 27670
gar_fias.as_houses: 665054
gar_fias.as_steads: 258363
gar_fias.as_apartments: 222581
gar_fias.as_rooms: 601
gar_fias.as_carplaces: 646
gar_fias.as_adm_hierarchy: 809351
gar_fias.as_mun_hierarchy: 794190
gar_fias.as_change_history: 2620890
gar_fias.as_addr_obj_division: 214
gar_fias.as_addr_obj_params: 326128
gar_fias.as_steads_params: 3296039
gar_fias.as_houses_params: 3839582
gar_fias.as_apartments_params: 590864
gar_fias.as_rooms_params: 693
gar_fias.as_carplaces_params: 645
gar_fias.as_normative_docs: 192138
